benchmarks/benchmarks/__init__.py

```diff
@@ -54,8 +53,22 @@ class TrackAddedMemoryAllocation:
             other_call()
         result = mb.addedmem_mb()
 
+    Attributes
+    ----------
+    RESULT_MINIMUM_MB : float
+                        The smallest result that should ever be returned, in
```

Comment:
The 'attributes' descriptions don't need to be indented so far.
See : https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring

Reply:
So it was a coincidence that the attribute descriptions aligned with the attribute types in that example? 🤣 They're both exactly one tab in!

Reply:
I'll reduce everything to a single tab, hang on...

Reply:
0301a8091bb616f5bd4997e8798a9f3079d0aa9b